### PR TITLE
Update for PHP 8

### PR DESCRIPTION
**Pull request by repat**
See #24

### CODE REVIEW

**Comment by repat** [comment]
image

-- https://scrutinizer-ci.com/g/beyondcode/helo-laravel/inspections/434e509d-e069-426e-ae36-242d54f15aef

Not sure what this means

File: composer.json [hunk 1]
```diff
@@ -16,14 +16,14 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": ">=7.3",
```
Comment (dacoto97) [inline composer.json:19]: "php": "^7.3|^8.0",

**Review by dacoto97 — CHANGES_REQUESTED** [review]
Add php 7.3 & 8.0

**Comment by devingray** [comment]
Hi @mpociot @repat 

This PR is old now and I would like to make PR for this to get PHP 8 working :) Would you like a new PR or to update here?

**Pull request closed without merging**